docs/topics/modules/approving_build_pipeline.adoc

```diff
@@ -3,34 +3,40 @@
 
 Several minutes after starting, your build pipeline is ready to be promoted to `Stage`.
 
-The build pipeline includes steps for a `Stage` build and a `Run` build. `Stage` is a production staging area to review and test changes before they finalizing them and staging the build on `Run`.
+The build pipeline includes steps for a `Stage` build and a `Run` build. `Stage` is a production staging area to review and test your application before finalizing it and staging the build on `Run`.
 
-To review the final changes in `Stage` and promote them to `Run`:
+To `Stage` your application and review:
```

Comment:
We should probably make a quick decision about referring to Stage and Run (environments) versus just saying 'to stage changes/your app'. I would recommend that if we use the proper noun for the environments, we should.not use them as verbs, i.e. either 'to view your application on 'Stage'' would be about the environment called stage while 'to stage your application' refers to generically staging an application somewhere. Thoughts, @rkratky and @preetic

Reply:
I have tried to keep it in sync with the other doc, but happy to change. Let me know.

Reply:
Agree with Misha: "to stage your app on Stage".